ironfish/src/rpc/routes/wallet/createTransaction.test.ts

```diff
@@ -240,8 +240,8 @@ describe('Route wallet/createTransaction', () => {
       ],
       mints: [
         {
```

Comment:
The current RPC usage via CLI is using plaintext input for these params, so I think these tests aren't representative.